tsl/src/compression/compression.c

```diff
@@ -623,7 +623,6 @@ index_scan_sequence_number(Relation table_rel, Oid index_oid, ScanKeyData *scank
 	int32 result = 0;
 	bool is_null;
 	Relation index_rel = index_open(index_oid, AccessShareLock);
-	RelationInitIndexAccessInfo(index_rel);
```

Comment:
Is this a relevant change?

Reply:
Yes, it is pull request in main, I wanted to check if it helps with sanitizer fails, As soon as it is pushed into main, I will cherrypick it into 2.9.x